angular-workspace/tsconfig.json

```diff
@@ -5,6 +5,24 @@
     "paths": {
       "@ni/nimble-angular": [
         "dist/ni/nimble-angular"
+      ],
```

Comment:
tsconfig paths supports wildcards, I think we just need:

```json
"paths": {
      "@ni/nimble-angular": [
        "dist/ni/nimble-angular"
      ],
      "@ni/nimble-angular/*": [
        "dist/ni/nimble-angular/*"
      ],
}
```

Reply:
Yes, you are correct. That makes this configuration much better!